distribution/dashboard/ui/src/constants/schedule.ts

```diff
@@ -0,0 +1,73 @@
+import { range } from 'lodash';
+
+export const SCHEDULE_TYPE_OPTIONS = [
+  {
+    label: '周',
+    value: 'Weekly',
+  },
+  {
+    label: '月',
+    value: 'Monthly',
+  },
+];
+
+export const WEEK_OPTIONS = [
+  {
+    label: '一',
+    fullLabel: '星期一',
+    value: 1,
+  },
+
+  {
+    label: '二',
+    fullLabel: '星期二',
+
+    value: 2,
+  },
+
+  {
+    label: '三',
+    fullLabel: '星期三',
+
+    value: 3,
+  },
+  {
+    label: '四',
+    fullLabel: '星期四',
+
+    value: 4,
+  },
+  {
+    label: '五',
+    fullLabel: '星期五',
+
+    value: 5,
+  },
+  {
+    label: '六',
+    fullLabel: '星期六',
+
+    value: 6,
+  },
+  {
+    label: '日',
+    fullLabel: '星期日',
+
+    value: 7,
+  },
+];
+
+export const WEEK_TEXT_MAP = new Map
```

Comment:
const myMap = new Map([
  ['key1', 'value1'],
  ['key2', 'value2'],
  ['key3', 'value3']
]);  
Would this be more intuitive？